Reference/V9-Config/InstallDefaultDataSettings/index.md

```diff
@@ -0,0 +1,111 @@
+---
+versionFrom: 9.0.0
+meta.Title: "Umbraco Install Default Data Settings"
+meta.Description: "Information on configuration allowing for the modification of default data installed in new projects"
+---
+
+# Install Default Data Settings
+
+When Umbraco is installed for the first time, it creates various default data.  These include a language, some data types and some media and member types.
+
+In certain setups you may want to take control over what is installed, and opt-out of the creation of certain items.
```

Comment:
In certain setups, you may want to take control over what is installed and opt-out